src/pkg/k8s/namespace.go

```diff
@@ -20,6 +20,12 @@ func (k *K8s) GetNamespaces(ctx context.Context) (*corev1.NamespaceList, error)
 	return k.Clientset.CoreV1().Namespaces().List(ctx, metaOptions)
 }
 
+// GetNamespace gets a namespace by name
+func (k *K8s) GetNamespace(name string) (*corev1.Namespace, error) {
```

Comment:
Can you pass the context in `editHelmResources()` into here, it seems to be the only place this is being called

Reply:
Whoops I meant to delete this function (it's actually not getting called at all anymore `editHelmResources()` calls `getNamespaces()` )